analyzers/src/SonarAnalyzer.Common/Extensions/CountComparisonResultExtensions.cs

```diff
@@ -0,0 +1,85 @@
+﻿/*
+ * SonarAnalyzer for .NET
+ * Copyright (C) 2015-2021 SonarSource SA
+ * mailto: contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+namespace SonarAnalyzer.Helpers
+{
+    public static class CountComparisonResultExtensions
+    {
+        public static bool EmptyOrNotEmpty(this CountComparisonResult comparison) =>
+            comparison == CountComparisonResult.Empty || comparison == CountComparisonResult.NotEmpty;
+
+        public static CountComparisonResult Count(this ComparisonKind comparison, int count) =>
+            comparison switch
+            {
+                ComparisonKind.Equals => Equals(count),
+                ComparisonKind.NotEquals => NotEquals(count),
+
+                ComparisonKind.GreaterThanOrEqual => GreaterThanOrEqual(count),
+                ComparisonKind.GreaterThan => GreaterThan(count),
+
+                ComparisonKind.LessThan => LessThan(count),
+                ComparisonKind.LessThanOrEqual => LessThanOrEqual(count),
+
+                _ => CountComparisonResult.None,
```

Comment:
The handler name is always the same as the enum member = regular, easy to follow. We don't need those empty lines as the logic doesn't differ.
```suggestion
                ComparisonKind.Equals => Equals(count),
                ComparisonKind.NotEquals => NotEquals(count),
                ComparisonKind.GreaterThanOrEqual => GreaterThanOrEqual(count),
                ComparisonKind.GreaterThan => GreaterThan(count),
                ComparisonKind.LessThan => LessThan(count),
                ComparisonKind.LessThanOrEqual => LessThanOrEqual(count),
                _ => CountComparisonResult.None,
```